{"uuid":"39c91dba-268d-4cdf-9516-68962ee8127f","historyId":"99ff556da8852dd5af64bd9e23aa2482","testCaseId":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.GradeServiceTest]/[method:getBySubmissionId_DataAccessException()]","testCaseName":"Get By Submission ID - Throws DatabaseServiceException on database error","fullName":"org.academo.academo.London.service.GradeServiceTest.getBySubmissionId_DataAccessException","labels":[{"name":"tag","value":"unit"},{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.GradeServiceTest]/[method:getBySubmissionId_DataAccessException()]"},{"name":"host","value":"Rockio"},{"name":"thread","value":"5188@Rockio.main(3)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"org.academo.academo.London.service.GradeServiceTest"},{"name":"testClass","value":"org.academo.academo.London.service.GradeServiceTest"},{"name":"testMethod","value":"getBySubmissionId_DataAccessException"},{"name":"suite","value":"Grade Service Implementation Tests"}],"links":[],"name":"Get By Submission ID - Throws DatabaseServiceException on database error","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1757414934911,"stop":1757414934913}